README.md

```diff
@@ -1 +1,109 @@
-# FreeCodeCamp-Portfolio
\ No newline at end of file
+# FreeCodeCamp-Portfolio🚀
+
+## Project Description 📝
+
+> 🌟 Welcome to Shogofa Developer's Portfolio! 🌟
+
+Embark on a journey through the realm of web development with me, Shogofa Developer. 🚀
+
+As a diligent web development enthusiast, I've dedicated numerous hours to refining my skills in remote development. From crafting sleek landing pages to mastering various projects, I've immersed myself in a wide array of tasks. My portfolio showcases a range of projects, each crafted with passion and attention to detail.
+
+Join me as I continue to pursue excellence in the dynamic world of web development! 💻✨
+
+## Live Link
+
+## Demo 📸
+
+![Project Image](Image/ph.png)
+
+## Technologies Used 🛠️
+
+- [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML)
+- [CSS](https://developer.mozilla.org/en-US/docs/Web/CSS)
+
+## Installation 💻
+
+To install and set up this project, simply follow these steps:
+
+1. **Clone the repository:**
+   ```bash
+   git clone git@github.com:shogof/Technical-Documentation-page.git
```

Comment:
- Add this project link not your technical documentation page

Reply:
OK Sir I do it now and push it again 
really sorry about that